resources/types/Pangenome.yaml

```diff
@@ -0,0 +1,21 @@
+global-object-type: Pangenome
```

Comment:
Which of these values are mutable immutables?

Reply:
They are all immutable. Has separate "id" attribute that is not being indexed.

Reply:
I thought some of the fields being indexed were object names, which are mutable immutables in this context?

Reply:
No, none of the currently indexed fields are object names

Reply:
I'm confused then:

>  MrCreosote 7 hours ago Owner
> Is this field the object name again?
> JamesJeffryes 7 hours ago Member
> Yes, it's what the people want...

Reply:
That was referring to RNASequenceSet. Thought the question was specific to the Pangenome object where this comment is. 

The "default_node_labels" in the Tree object may refer to genome names which are mutable (though most will be reference genomes or feature names which are immutable)

Reply:
I was asking about all the types here. 

Anyway, is the RSS id and the node labels in the tree the only mutable immutables then?

Thinking further, I get the use case where you extract the id from a dependent object (like Model extracting from Genome) so you can find objects based on the dependent object name (even though that name may be out of date).

I *don't* understand the use case of indexing the object's own name via the spec. The name automatically gets indexed, so if the name isn't changed it buys nothing. If the name is changed, it just means the object always shows up in searches for its original name as well as its real name, which doesn't seem like desired behavior.

Reply:
You are right, the RSS id does not make sense alone. I've used the sample_set_description instead and added a few other indexed fields. This should make node_lables the only potentially mutable type